app/src/main/java/org/traccar/manager/DevicesFragment.java

```diff
@@ -53,10 +63,41 @@ public View getView(int position, View convertView, ViewGroup container) {
         }
     }
 
+    @Nullable
+    @Override
+    public View onCreateView(final LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = super.onCreateView(inflater, container, savedInstanceState);
+
+        FrameLayout root = (FrameLayout)view;
+        final Context context = getActivity();
+        FloatingActionButton addButton = new FloatingActionButton(context);
```

Comment:
In the 5830c4e06c0418c42b195f61e2a2087ab7efe51f version it's there (under "Unable to press add button fix" commit)